x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.ts

```diff
@@ -268,6 +273,7 @@ export function registerTransactionDurationRuleType({
               [SERVICE_NAME]: serviceName,
               ...getEnvironmentEsField(environment),
               [TRANSACTION_TYPE]: transactionType,
+              [TRANSACTION_NAME]: ruleParams.transactionName,
```

Comment:
I can't comment on the line (244), but I think the alert ID needs to be updated as well

Reply:
@dgieselaar I have updated alert ID [in this PR](https://github.com/elastic/kibana/pull/155001).

Reply:
Are you referring to this issue right? https://github.com/elastic/kibana/issues/154818

The AO addressed the bug in this PR 
https://github.com/elastic/kibana/pull/155001/files#diff-168f6c518c41cc2f500c7b116f10d038808dc824f91c21baf72075f7d6bd4858R265

Do you think it would be better to fix it in this PR instead of waiting the other PR to be merged?